infrastructure/docker/build/docker-compose.yml

```diff
@@ -76,3 +76,10 @@ services:
       context: ../../..
     volumes:
       - ../../..:/trafficcontrol:z
+  traffic_weasel:
```

Comment:
can you change this?  looks like `traffic_weasel` is another component of traffic_control,  which could be confusing.  It's just a tool being run on the code.   Call it `license_check` or something similar.  And a nit -- add a blank line above the name to make sections more visible.